api/nodes/views.py

```diff
@@ -232,12 +236,38 @@ def get_default_odm_query(self):
         query = base_query & permission_query
         return query
 
-    # overrides ListCreateAPIView
+    # overrides ListBulkCreateJSONAPIView, BulkUpdateJSONAPIView
     def get_queryset(self):
-        query = self.get_query_from_request()
-        return Node.find(query)
+        # For bulk requests, queryset is formed from request body.
+        if is_bulk_request(self.request):
```

Comment:
I didn't think GET bulk requests should be a thing.  With a GET request, there's nothing in the request body, so it's not a bulk request.  If you try to do a GET bulk request, it returns an empty list.